packages/rancher-monitoring/rancher-monitoring/generated-changes/patch/templates/prometheus/servicemonitor.yaml.patch

```diff
@@ -1,16 +1,9 @@
 --- charts-original/templates/prometheus/servicemonitor.yaml
 +++ charts/templates/prometheus/servicemonitor.yaml
-@@ -35,10 +35,20 @@
-     bearerTokenFile: {{ .Values.prometheus.serviceMonitor.bearerTokenFile }}
+@@ -38,6 +38,16 @@
+     {{- if .Values.prometheus.serviceMonitor.metricRelabelings }}
+     metricRelabelings: {{- tpl (toYaml .Values.prometheus.serviceMonitor.metricRelabelings | nindent 6) . }}
      {{- end }}
-     path: "{{ trimSuffix "/" .Values.prometheus.prometheusSpec.routePrefix }}/metrics"
--{{- if .Values.prometheus.serviceMonitor.metricRelabelings }}
-     metricRelabelings:
--{{ tpl (toYaml .Values.prometheus.serviceMonitor.metricRelabelings | indent 6) . }}
--{{- end }}
-+    {{- if .Values.prometheus.serviceMonitor.metricRelabelings }}
-+    {{ tpl (toYaml .Values.prometheus.serviceMonitor.metricRelabelings | indent 6) . }}
-+    {{- end }}
```

Comment:
Metric relabellings should still be included here, not just the cattle system ones

Reply:
Fixed in upstream: https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/servicemonitor.yaml#L38-L40